Source/CLLocationManager+LambdaKit.swift

```diff
@@ -91,7 +91,6 @@ extension CLLocationManager: CLLocationManagerDelegate {
     /// Request the current location which is reported in the completion handler.
     ///
     /// - parameter completion: A closure that will be called with the device's current location.
-    @available(iOS 9, watchOS 2, *)
```

Comment:
Shouldn't we leave the iOS 9 part since the deployment target of the library is still 8.0?